core/src/v2/models.rs

```diff
@@ -107,6 +107,18 @@ pub struct Api<P, R, S> {
     pub parameters: BTreeMap<String, P>,
     #[serde(default = "BTreeMap::new", skip_serializing_if = "BTreeMap::is_empty")]
     pub responses: BTreeMap<String, R>,
+    #[serde(
+        default,
+        rename = "securityDefinitions",
+        skip_serializing_if = "BTreeMap::is_empty"
+    )]
+    pub security_definitions: BTreeMap<String, SecurityScheme>,
+    #[serde(default, skip_serializing_if = "BTreeMap::is_empty")]
```

Comment:
The method should now be `Vec::is_empty`

Reply:
Is there a way to run these changes locally first before push to github?

Reply:
I somehow missed this. Yes, you can use `make` to build and test the repository. Please check the `Makefile` in the repo root for building and testing commands.